CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Unreleased changes
 
 * Add wasm message types ([#133](https://github.com/provenance-io/provwasm/issues/133))
+* Bump to Provenance 1.17.1 ([#135](https://github.com/provenance-io/provwasm/issues/135))
+* Add ibc message types ([#136](https://github.com/provenance-io/provwasm/issues/136))
```

Comment:
Should we include a note about the optimizer?

Reply:
fixed